orttraining/orttraining/core/graph/optimizer_builder.h

```diff
@@ -59,6 +64,8 @@ class OptimizerBuilder {
    * @param gradient_norm_finite_argdef (Optional) The ArgDef indicates whether
             the passed-in gradient norm is finite.
    * @param opt_config The optimizer configuration.
+   * @param opt_graph_config The config for building optimizer graph,
+   *                           of type OptimizerGraphConfig
```

Comment:
nit: it seems unnecessary to name the type of the parameter. we have a static type system :)